jenkins/e2e-image/e2e-runner.sh

```diff
@@ -126,6 +126,15 @@ function call_get_kube() {
     export KUBERNETES_RELEASE_URL
     KUBERNETES_SKIP_CONFIRM=y KUBERNETES_SKIP_CREATE_CLUSTER=y KUBERNETES_DOWNLOAD_TESTS=y \
       "$(dirname "${0}")/get-kube.sh"
+    if [[ ! -x kubernetes/cluster/get-kube-binaries.sh ]]; then
```

Comment:
Maybe print a warning that we shouldn't hit this path except when we are supposed to hit this path?